react/features/invite/components/dial-in-summary/NumbersList.web.js

```diff
@@ -41,76 +41,144 @@ class NumbersList extends Component<Props> {
      * @returns {ReactElement}
      */
     render() {
-        const { numbers, t } = this.props;
+        const { numbers } = this.props;
 
         return (
-            <table className = 'dial-in-numbers-list'>
-                <thead>
-                    <tr>
-                        { Array.isArray(numbers)
-                            ? null
-                            : <th>{ t('info.country') }</th> }
-                        <th>{ t('info.numbers') }</th>
-                    </tr>
-                </thead>
-                <tbody className = 'dial-in-numbers-body'>
-                    { Array.isArray(numbers)
-                        ? numbers.map(this._renderNumberRow)
-                        : this._renderWithCountries(numbers) }
-                </tbody>
-            </table>);
+            <div className = 'dial-in-numbers-list'>
+                { this._renderWithCountries(numbers) }
+            </div>);
     }
 
     /**
      * Renders rows of countries and associated phone numbers.
      *
-     * @param {Object} numbersMapping - An object with country names as keys
-     * and values as arrays of phone numbers.
+     * @param {Object|Array<Object>} numbersMapping - An object with country
+     * names as keys and values as arrays of phone numbers.
      * @private
      * @returns {ReactElement[]}
      */
-    _renderWithCountries(numbersMapping: Object) {
-        const rows = [];
+    _renderWithCountries(
+            numbersMapping: { numbers: Array<string> } | Array<Object>) {
+        const { t } = this.props;
+        let numbers;
+
+        if (Array.isArray(numbersMapping)) {
+            numbers = numbersMapping.reduce(
+                (resultNumbers, number) => {
+                    const countryName
+                        = t(`countries:countries.${number.countryCode}`);
+
+                    if (resultNumbers[countryName]) {
+                        resultNumbers[countryName].push(number);
+                    } else {
+                        resultNumbers[countryName] = [ number ];
+                    }
+
+                    return resultNumbers;
+                }, {});
+        } else {
+            numbers = {};
+
+            for (const [ country, numbersArray ]
+                of Object.entries(numbersMapping.numbers)) {
+                if (!Array.isArray(numbersArray)) {
+                    return;
+                }
```

Comment:
Do you know if this early return is on purpose? I actually think it's a bug I might have introduced while trying to fix flow. Maybe instead of a return it should be a continue.

Reply:
I will check it, yeah I left it from the previous code...

Reply:
Fixed.